From e540b5542eef23a7b600e774bd65d1da37d9e0a8 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Mon, 11 Dec 2017 19:14:57 +0000
Subject: [PATCH 2/2] Use overlay2 as the default storage driver

overlay was causing issues with the 4.14 kernel resulting in memory
exhaustion and symlink resolution problems. Switching to overlay2
resolves these issues at a slight performance cost.
---
 components/packaging/rpm/systemd/docker.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/packaging/rpm/systemd/docker.service b/components/packaging/rpm/systemd/docker.service
index 6c60646..4123588 100644
--- a/components/packaging/rpm/systemd/docker.service
+++ b/components/packaging/rpm/systemd/docker.service
@@ -9,7 +9,7 @@ Type=notify
 # the default is not to use systemd for cgroups because the delegate issues still
 # exists and systemd currently does not support the cgroup feature set required
 # for containers run by docker
-ExecStart=/usr/bin/dockerd
+ExecStart=/usr/bin/dockerd --storage-driver=overlay2
 ExecReload=/bin/kill -s HUP $MAINPID
 # Having non-zero Limit*s causes performance problems due to accounting overhead
 # in the kernel. We recommend using cgroups to do container-local accounting.
-- 
2.15.0

